### PR TITLE
Minor refactor and Changelog!

### PR DESCRIPTION
**Pull request by s-taylor**
## Refactor
Main change is using `_.times` instead of reduce and just returning an array to later .join

## Changelog
Now changelogs with every version!

### CODE REVIEW

**Comment by s-taylor** [comment]
i was bored ok...

File: index.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+const _ = require('lodash');
 const colors = require('colors'); // eslint-disable-line no-unused-vars
 // this ~crazy-scary~ _amazing_ package mutates Strings globally
 const copyPaster = require('copy-paste');
 
-const supposedDate = process.argv[2];
-let date = new Date(supposedDate);
-if (!date.getDate()) date = new Date();
+const target = process.argv[2];
+const date = target ? new Date(target) : target;
```
Comment (one19) [inline index.js]: this breaks the no-input use case.

calling `todo-month` from the command line with no arguments should return your current month.

Comment (s-taylor) [inline index.js]: errr crap that last part is supposed to be `new Date`

File: index.js [hunk 2]
```diff
@@ -18,20 +18,25 @@ const DAY_NAMES = [
...
+const fmt = str => `0${str}`.slice(-2);
 const weekday = (year, month, day) => new Date(year, month, day).getDay();
-const daysInMonth = (year, month) =>
-  new Date(year, month, 0).getDate();
-const dayAbbrevAppender = dayNumber =>
-  dayNumber === 0 || dayNumber === 6 ? DAY_NAMES[dayNumber].toUpperCase() :
-    DAY_NAMES[dayNumber].slice(0, 2);
+const daysInMonth = (year, month) => new Date(year, month, 0).getDate();
+const dayAbbrevAppender = dayNumber => (
+  [0, 6].contains(dayNumber)
```
Comment (one19) [inline index.js]: this should be `includes`

File: index.js (hunk 1 above)
Comment (s-taylor) [inline index.js]: @one19 ok try now

File: index.js (hunk 2 above)
Comment (s-taylor) [inline index.js]: yerp fixed that also

**Pull request merged**